packages/phone-number-privacy/signer/package.json

```diff
@@ -38,6 +38,7 @@
   },
   "dependencies": {
     "@celo/base": "^5.0.3-dev",
+    "blind-threshold-bls": "npm:@celo/blind-threshold-bls@1.0.0-beta",
```

Comment:
why not refactoring to use `@celo/blind-threshold-bls` ?

Reply:
that caused an error because a resolution is still required as some inner deps still would use blind direct